docs/modules/monitoring/con-proxy-integrating-micrometer.adoc

```diff
@@ -88,6 +88,20 @@ micrometer:
 
 |===
 
+== Pause detector
```

Comment:
We should probably give a little context here before saying see the micrometer docs for full details or something similar.